backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/ViewToolService.java

```diff
@@ -949,15 +949,23 @@ protected List<? extends Object> getChildrenWithInstancesOf(Object selectionDial
      */
     protected boolean isExposable(Element element) {
         boolean isExposable = false;
-        var declaredName = element.getDeclaredName();
         if (this.utilService.isStandardDoneAction(element) || this.utilService.isStandardStartAction(element)) {
             isExposable = false;
         } else {
-            isExposable = declaredName != null && !declaredName.isBlank();
+            isExposable = hasIdentifier(element);
```

Comment:
hasIdentifier => this.hasIdentifier

Reply:
Done. I fixed my IDE so it will give me an error whith missing this.